### PR TITLE
sample: add sample script for launching xenomai RTVM

### PR DESCRIPTION
**Pull request by totoroow**
Add a launch script for xenomai RTVM.

Tracked-On: #4514
Signed-off-by: Tw <wei.tan@intel.com>
Reviewed-by: Binbin Wu <binbin.wu@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: devicemodel/samples/nuc/launch_xenomai.sh [hunk 1]
```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
```
Comment (gvancuts) [inline devicemodel/samples/nuc/launch_xenomai.sh]: Please add the copyright and license information (check the other launch script files for examples)

File: devicemodel/samples/nuc/launch_xenomai.sh [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+	# extract the kernel and install kernel modules
+	bzImage=/mnt/$(tar xzvf $kernel -C /mnt --dereference --exclude=vmlinux* | grep vmlinuz)
+	cp $bzImage /tmp/bzImage.xenomai
+
+	# umount disk
+	umount /mnt
+
+	# detect partition information
+	disk_part_nr=$(basename $disk| sed 's/.*\([0-9]$\)/\1/')
+	if grep -q "nvme" "$disk"; then
```
Comment (gvancuts) [inline devicemodel/samples/nuc/launch_xenomai.sh]: should we also account for people usign eMMC (i.e. `mmcblk`)?

Comment (totoroow) [reply devicemodel/samples/nuc/launch_xenomai.sh]: Yes, support for more types of storage (e.g. eMMC) will be great. As this is just a sample script for users to refer to and NVMe and SATA are two major types we support now. Let's keep it simple for now, maybe we could add more later.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**